install/upgrade-clickhouse.sh

```diff
@@ -28,6 +28,10 @@ if [[ -n "$(docker volume ls -q --filter name=sentry-clickhouse)" ]]; then
     $dcb --build-arg BASE_IMAGE=altinity/clickhouse-server:23.3.19.33.altinitystable clickhouse
     $dc up -d clickhouse
     wait_for_clickhouse
+    $dc down clickhouse
+    $dcb --build-arg BASE_IMAGE=altinity/clickhouse-server:24.3.5.47.altinitystable clickhouse
```

Comment:
I think you meant `23.8.11.2923.8.11.29` here?

Reply:
Yes. I fixed